### PR TITLE
Add restore for users and groups to V1.0

### PR DESCRIPTION
**Pull request by PatAltimore**

### CODE REVIEW

File: api-reference/v1.0/api/directory_deleteditems_list.md [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+|Permission type      | Permissions (from least to most privileged)              |
+|:--------------------|:---------------------------------------------------------|
+|Delegated (work or school account) | Group.Read.All, Group.ReadWrite.All, Directory.Read.All, Directory.AccessAsUser.All |
+|Delegated (personal Microsoft account) | Not supported.    |
+|Application | Group.Read.All, Group.ReadWrite.All, Directory.Read.All |
+
+## HTTP request
+<!-- { "blockType": "ignored" } -->
+```http 
+GET /directory/deleteditems/Microsoft.Graph.Group
```
Comment (dkershaw10) [inline api-reference/v1.0/api/directory_deleteditems_list.md]: While this work with initial uppercase for the types, you should make these all lowercase
`GET /directory/deleteditems/microsoft.graph.group`

This should be done for all topics too.  Please fix this in the Beta docs (but you can do this as a separate bug fix PR)

File: concepts/changelog.md [hunk 2]
```diff
@@ -4,6 +4,12 @@ This changelog covers what's changed in Microsoft Graph, including the v1.0 and
 
 For details about known issues with Microsoft Graph APIs, see [Known issues](known_issues.md).
 
+### Directory APIs
+
+| **Change type** | **Version** | **Description**                          |
+| :-------------- | :---------- | :--------------------------------------- |
+| Addition        | V1.0        | Added support for [restoring and permanently deleting users and groups](https://graph.microsoft.io/en-us/docs/api-reference/v1.0/resources/directory). |
```
Comment (dkershaw10) [inline concepts/changelog.md:13]: I wonder whether you need to add "listing deleted items" within your square brackets?

File: api-reference/v1.0/api/directory_deleteditems_get.md [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+|:--------------------|:---------------------------------------------------------|
+|Delegated (work or school account) | User.Read.All, User.ReadWrite.All, Directory.Read.All, Directory.ReadWrite.All, Directory AccessAsUser.All |
+|Delegated (personal Microsoft account) | Not supported. |
+|Application | User.Read.All, User.ReadWrite.All, Directory.Read.All, Directory.ReadWrite.All |
+
+### For groups:
+
+|Permission type      | Permissions (from least to most privileged)              |
+|:--------------------|:---------------------------------------------------------|
+|Delegated (work or school account) | Group.Read.All, Group.ReadWrite.All, Directory.Read.All, Directory.AccessAsUser.All |
```
Comment (dkershaw10) [inline api-reference/v1.0/api/directory_deleteditems_get.md:22]: This is missing Directory.ReadWrite.All (as is the app only permission).

**Comment by lleonard-msft** [comment]
The work in this PR was merged in this [PR instead](https://github.com/microsoftgraph/microsoft-graph-docs/pull/2387).

**Pull request closed without merging**